### DISCUSSION TITLE
Github student pack for my account is ended too early

### DISCUSSION THREAD

**Discussion by stackoverprof** [community/community#42245] [GitHub Education]
I have obtained github student pack Nov 10 2020
And should still be active until July 2024, according to the student ID card submitted

But just this month, my github student pack benefit is ended. 
Which is too early... and GitHub is starting to billing me if I wished to keep using the services.

I have raised a ticket to the GitHub support but then they redirected me to this discussion page to get the support instead.

I need help, thanks in advance.

Account Name: stackoverprof https://github.com/stackoverprof

![Screenshot 2022-12-20 at 19 03 46](https://user-images.githubusercontent.com/58414739/208787229-db6cc723-5b70-43d1-a630-5f88d601ad42.png)
![Screenshot 2022-12-20 at 18 57 34](https://user-images.githubusercontent.com/58414739/208787240-33b89e54-6442-4257-988e-b9ad29057fbd.png)

**Comment by xixilaladsds** [comment]
The same with you. I have successfully registered my student package  this August but I can't use the profit recently. My ticket is also closed and redirect me to find help in the community.

**Reply by stackoverprof** [reply]
What have you done so far? Is there any solution from the community?

**Reply by xixilaladsds** [reply]
I just upgrade my copilot and then I lost it. When it happened, I submitted my ticket to the Github Support Center as soon as possible but nobody answered. I finally received a replay to be told to ask question in the community util yesterday.

I don't know what to do next.

**Comment by samus-aran** [comment]
Hi @stackoverprof & @xixilaladsds 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.
To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.

**Comment by MuhammadAliAsgher** [comment]
same issue occurred
https://github.com/orgs/community/discussions/135252#discussion-7030574

**Reply by samus-aran** [reply]
Hi @MH-11103 👋 

Try not to post in stale discussions as you have already created a separate discussion for others to assist you with.

We're locking this discussion because it has not had recent activity. Thank you for helping us maintain a productive and tidy community for all our members.
